http_parser.c

```diff
@@ -472,6 +483,92 @@ static struct {
 
 int http_message_needs_eof(const http_parser *parser);
 
+const char* find_crlf(const char* p, const char* data, size_t len);
+
+
+#if defined(__SSE2__) && defined(__GNUC__) 
+
+#include <emmintrin.h>
```

Comment:
Nice work with this PR!

It's trivial to make this work on Windows also, just change these two lines to:

```cpp
#if defined(_MSC_VER) // SSE2 is baseline
#include <intrin.h>
#define __builtin_ctz _tzcnt_u32
#elif defined(__SSE2__) && defined(__GNUC__) 
#include <emmintrin.h>
#endif

#if defined(_MSC_VER) || (defined(__SSE2__) && defined(__GNUC__))
```